rktl_game/nodes/game_manager

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""Use ball location to determine when a goal is scored.
+License:
+  BSD 3-Clause License
+  Copyright (c) 2022, Autonomous Robotics Club of Purdue (Purdue ARC)
+  All rights reserved.
+"""
+
+
+
+# ROS
+import rospy
+from std_msgs.msg import Bool, Header
+from nav_msgs.msg import Odometry
+from rktl_msgs.msg import Score, MatchStatus
+from std_srvs.srv import Empty, EmptyResponse
+
+class ScoreKeeper():
+    def __init__(self):
+        rospy.init_node('score_keeper')
+
+        self.orange_score = 0
+        self.blue_score = 0
+        self.orange_goal = (rospy.get_param('/field/length', 1) - 0.15) / 2 
```

Comment:
What is going on here? Also, it is preferred to not pass in a default for our global params so that it throws an error if it isn't found. This is mainly because the launch file that launches the global params is pretty far from the one that launches the node, so it is easy to accidentally forget to launch them

Reply:
I removed the defaults, idk what you didn't know what was going on but if it's why i'm subtracting 15 centimeters it's because the field is never put exactly in the right location so we need the ball to be outside the field by a few inches to keep it from calling a bounce off the wall a goal

Reply:
I'd make the 0.15 a parameter rather than hardcoding. Something private. But yes, that is what I was asking.